### ISSUE TITLE
Update Footer to have grid with sections

### ISSUE THREAD

**Issue by kjaymiller** [BlackPythonDevs/blackpythondevs.github.io#470]
We have some information that is currently not on the website that we can add.

#### To Fix

We should restructure the footer to a grid layout.

#### About Column

- About BPD
- Leadership
- Mission
- Code of Conduct

#### Support Column

- Support

#### Programs and Events Column

- Events
- Meetup
- Programs

#### Community Column

- Community
- Social Links

**Comment by Selasi3** [comment]
I will pick it up

**Comment by TimOsahenru** [comment]
Hello @kjaymiller, I think we should include scenarios like this to our CONTRIBUTING/MAINTAINERS file. "... contributors should not be assigned to a new issue if they have an open PR still in review, except in scenarios where the issues are exclusively related to the existing PR", something like that

**Comment by kjaymiller** [comment]
I'm okay with @Selasi3 making the changes but they need to be on a separate PR.

**Comment by Selasi3** [comment]
I would like to work on this task.

**Comment by TimOsahenru** [comment]
CC: @kjaymiller

**Comment by TimOsahenru** [comment]
Hi @Selasi3, just following up to check if there are any updates on this issue. Thanks

**Comment by kjaymiller** [comment]
Reopening due to lack of communication. @TimOsahenru please proceed to work on this.

**Comment by TimOsahenru** [comment]
> Reopening due to lack of communication. @TimOsahenru please proceed to work on this.

Thanks Jay

**Comment by TimOsahenru** [comment]
> We have some information that is currently not on the website that we can add.
> 
> #### To Fix
> We should restructure the footer to a grid layout.
> 
> #### About Column
> * About BPD
> * Leadership
> * Mission
> * Code of Conduct
> 
> #### Support Column
> * Support
> 
> #### Programs and Events Column
> * Events
> * Meetup
> * Programs
> 
> #### Community Column
> * Community
> * Social Links

On the about column, shouldn't it be a short about on BPD and then maybe have a quick link for
> * About BPD
> * Leadership
> * Mission
> * Code of Conduct. So, something like this
> #### About Column
Black Python Devs was created by its founder Jay Miller after seeing a trend of the same handful of Black developers speaking at major conferences, taking leadership positions, and dealing with the same challenges towards burnout.    
> 
> #### Quick link
> * About BPD
> * Leadership
> * Mission
> * Code of Conduct
>

**Comment by kjaymiller** [comment]
I agree but I think it would be better to include basic organizational information.

> Black Python Devs is a US Non-Profit Organization under the fiscal hosting of the GNOME Foundation. The foundation aims to expand access to Python communities and resources to communities of Black Python Developers around the world.
